components/server/ee/src/monitoring-endpoint-ee.ts

```diff
@@ -9,18 +9,26 @@ import { WorkspaceHealthMonitoring } from "./workspace/workspace-health-monitori
 import { TraceContext } from "@gitpod/gitpod-protocol/lib/util/tracing";
 import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
 import { injectable, inject } from "inversify";
-import { register } from "../../src/prometheus-metrics";
+import { registerServerMetrics } from "../../src/prometheus-metrics";
+import * as prometheusClient from "prom-client";
+import { registerDBMetrics } from "@gitpod/gitpod-db/lib";
 
 @injectable()
 export class MonitoringEndpointsAppEE extends WorkspaceHealthMonitoring {
     @inject(WorkspaceHealthMonitoring) protected readonly workspaceHealthMonitoring: WorkspaceHealthMonitoring;
 
     public create(): express.Application {
+        const registry = prometheusClient.register;
+
+        prometheusClient.collectDefaultMetrics({ register: registry });
+        registerDBMetrics(registry);
```

Comment:
This change was needed to keep the metrics defined in the `gitpod-db` package, but bind it to server.